File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int  ch[]=new int[26];
        int l1=s.length();
        int l2=t.length();
        if(l1!=l2)return false;
        for(int i=0;i<l1;i++)
        {
            ch[s.charAt(i)-'a']++;
        }
        for(int i=0;i<l2;i++)
        {
            ch[t.charAt(i)-'a']--;
        }
        for(int i=0;i<26;i++)
        {
            if(ch[i]!=0)
            return false;
        }
        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.c
#include<string.h>
bool isAnagram(char* s, char* t) 
{
    if(strlen(s)==strlen(t))
    {
         int ar[26],c,i,r=0;
         for(int i=0;i<26;i++)
         ar[i]=0;
         for(i=0;i<strlen(s);i++)
         {
            c=((int)s[i])-97;
             ar[c]++;
         }
         for(i=0;i<strlen(s);i++)
         {
            c=((int)t[i])-97;
             ar[c]--;
         }
        for(i=0;i<26;i++)
       { if(ar[i]!=0)
        r++;}
        if(r==0)
        return true;
        else
        return false;
        
    }
    else 
        return false;
}